src/Microsoft.OpenApi/Exceptions/OpenApiReaderException.cs

```diff
@@ -39,7 +39,7 @@ public OpenApiReaderException(string message, ParsingContext context) : base(mes
         /// </summary>
         /// <param name="message">Plain text error message for this exception.</param>
         /// <param name="node">Parsing node where error occured</param>
-        public OpenApiReaderException(string message, JsonNode node) : base(message)
+        public OpenApiReaderException(string message, JsonNode? node) : base(message)
```

Comment:
we're not doing anything with the node parameter, remove the constructor entirely